Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -199,6 +199,20 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
             };
             resultCodeActions.push(vscodeCodeAction);
         });
+
+        const includeHeaderCodeAction: vscode.CodeAction = {
+            title: "Add include header",
+            command: {
+                title: "Add include header for symbol",
+                command: "C_Cpp.AddIncludeHeader",
+                arguments: [params.uri, range, token]
+            },
+            edit: undefined,
+            kind: vscode.CodeActionKind.QuickFix
+        };
+
+        resultCodeActions.push(includeHeaderCodeAction);
```

Comment:
I couldnt find a better way to do that. Therefore add it by default

Reply:
Hi @M1Z3 .  Thanks for contributing this.  We'd like to do this.  As you aluded to here, we may be able to determine whether the code action is needed based on IntelliSense results, to avoid showing it when not needed.  So, we may do additional work (possibly in the native, closed-source part of the extension) and may not necessarily incorporate exactly what you've provided.

Reply:
I totally agree. So how we are going to merge this? Do you want to change the IntelliSense implementation before merging this PR?